extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { memo } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { STATE_ERROR } from '../constants';
+
+const TrackError = memo( ( { link } ) => (
+	<div className="podcast-player__episode-error">
+		{ __( 'Episode unavailable', 'jetpack' ) }{ ' ' }
+		{ link && (
+			<span>
+				{ '(' }
+				<a href={ link } rel="noopener noreferrer nofollow" target="_blank">
+					{ __( 'Open in a new tab', 'jetpack' ) }
+				</a>
+				{ ')' }
+			</span>
+		) }
+	</div>
+) );
+
+const Track = memo( ( { track, isActive, isError, selectTrack, index } ) => {
+	return (
+		<li
+			className={ classnames( 'podcast-player__episode', {
+				'is-active': isActive,
+			} ) }
+		>
+			<a
+				className="podcast-player__episode-link"
+				href={ track.link }
+				role="button"
+				aria-pressed="false"
+				onClick={ e => {
+					// Prevent handling clicks if a modifier is in use.
+					if ( e.shiftKey || e.metaKey || e.altKey ) {
+						return;
+					}
+
+					// Prevent default behavior (opening a link).
+					e.preventDefault();
+
+					// Select track.
+					selectTrack( index );
+				} }
+				onKeyDown={ e => {
+					// Only handle the Space key.
+					if ( event.key !== ' ' ) {
+						return;
+					}
+
+					// Prevent default behavior (scrolling one page down).
+					e.preventDefault();
+
+					// Select track.
+					selectTrack( index );
+				} }
+			>
+				<span className="podcast-player__episode-status-icon"></span>
+				<span className="podcast-player__episode-title">{ track.title }</span>
```

Comment:
It seems that we may need to do some encoding work for the data displayed here. Try with `http://feeds.feedburner.com/lerendezvousjeux` for example, and you'll see an `&` displayed as `&amp;`

Reply:
Would you mind if we separated this out? It could be more tricky to handle in an already big PR. The current version, despite rendering entities poorly, is definitely safe to use - there is no way malicious HTML can be rendered or anything like it.

Reply:
Sure thing.